templates/vue/src/components/modals/NodeModal/forms/ContentForm/MultipleChoiceForm.vue

```diff
@@ -0,0 +1,274 @@
+          Add a choice
+        </b-button>
+        <p class="message">
+          Tick any options that should be selected by default in the question
+        </p>
+      </b-form-group>
+
+      <b-form-group v-else-if="!multipleAnswerSelected">
+        <sortable-list
+          v-model="choiceRowsRadio"
+          lockAxis="y"
+          :useDragHandle="true"
+          @input="updateOrderingRadioArray"
+        >
+          <b-form-checkbox-group
+            v-model="preSelectedRadioOptions"
+            class="radioGroup"
+          >
+            <choice-row
+              v-for="(choiceRow, index) in choiceRowsRadio"
+              :key="choiceRow.id"
+              :data-qa="`choicerow-radio-${choiceRow.id}`"
+              :item="choiceRow"
+              placeholder="answer option text"
+              :index="index"
+              :question="question"
+              :multipleAnswerSelected="multipleAnswerSelected"
+              :useImages="useImages"
+              :isDisabled="preSelectedRadioOptions.length > 0"
+              :selectedRadioChoice="preSelectedRadioOptions[0]"
+              :removeButtonDisabled="isRemoveButtonDisabled"
+              @remove="removeChoiceRowRadio(index, choiceRow)"
+            >
+              >
+            </choice-row>
+          </b-form-checkbox-group>
+        </sortable-list>
+        <b-button
+          class="addButton"
+          variant="primary"
+          squared
+          @click="addNewChoiceRadio"
+        >
+          Add a choice
+        </b-button>
+        <p class="message">
+          Tick any options that should be selected by default in the question
+        </p>
+      </b-form-group>
+    </b-form-group>
+  </div>
+</template>
+
+<script>
+import ChoiceRow from "./ChoiceRow"
+import { ContainerMixin } from "vue-slicksort"
+const SortableList = {
+  mixins: [ContainerMixin],
+  template: `
+    <ul class="list">
+      <slot />
+    </ul>
+  `,
+}
+export default {
+  components: {
+    ChoiceRow,
+    SortableList,
+  },
+  props: {
+    question: {
+      type: Object,
+      required: true,
+    },
+    multipleAnswerSelected: {
+      type: Boolean,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      preSelectedCheckBoxOptions: [],
+      preSelectedRadioOptions: [],
+      useImages: false,
+      choiceRows: [
+        {
+          id: 1,
+          imageurl: "",
+          value: "",
+        },
+        {
+          id: 2,
+          imageurl: "",
+          value: "",
+        },
+        {
+          id: 3,
+          imageurl: "",
+          value: "",
+        },
+      ],
+      nextChoiceRowId: 4,
+      choiceRowsRadio: [
+        {
+          id: 50,
+          imageurl: "",
+          value: "",
+        },
+        {
+          id: 51,
+          imageurl: "",
+          value: "",
+        },
+        {
+          id: 52,
+          imageurl: "",
+          value: "",
+        },
+      ],
+      nextChoiceRowRadioId: 53,
```

Comment:
As mentioned, let's use the same data structure for both radio and checkboxes. So we won't need these.